pkg/catalog/catalog.go

```diff
@@ -9,55 +9,68 @@ import (
 )
 
 // List lists all the available component types
-func List(client *occlient.Client) ([]string, error) {
+func List(client *occlient.Client) (finalList map[string][]string, err error) {
```

Comment:
please don't use named return values. It makes it hard to read :-(

Reply:
sure